source/Xunit.Gherkin.Quick/CoreModel/Arguments/PrimitiveTypeArgument.cs

```diff
@@ -26,7 +27,7 @@ public override void DigestScenarioStepValues(string[] argumentValues, StepArgum
             if (argumentValues.Length <= _index)
                 throw new InvalidOperationException($"Cannot extract value for parameter `{_parameterInfo.Name}` at index {_index}; only {argumentValues.Length} parameters were provided. Method `{_parameterInfo.Member.Name}`.");
 
-            Value = Convert.ChangeType(argumentValues[_index], _parameterInfo.ParameterType);
+            Value = Convert.ChangeType(argumentValues[_index], _parameterInfo.ParameterType, CultureInfo.InvariantCulture);
```

Comment:
Good, since currently the framework is not globalized. This will avoid cultural differences.

Reply:
Can you please add a unit test for this too?